### ISSUE TITLE
[QUESTION] How to speeding up nested grid model run?

### ISSUE THREAD

**Issue by dorosieun** [geoschem/geos-chem#924]
### Ask a question about GEOS-Chem:
Hi :>
I am currently attempting a nested model run for Asia with 0.25x0.3125 resolution.
My domain for simulation is as follows.
lon : 70 - 150E
lat : -7 - 50N
As a result of my test, it takes about 12h/day to run for the domain.
I've seen how to speed up as suggested in #875, and that method has already been applied by me.
Is there any other technique to speed up?

I'm running GEOS-Chem with PBS scheduler and available number of threads is 36 per 1 nodes.

**Comment by dorosieun** [comment]
Thanks @yantosca.
I keep looking the discussion #875. 

@msulprizio said,
> I would therefore recommend that users revert to the 10/20 minute (600/1200 s) timesteps in input.geos for MERRA-2 nested-grid simulations if they are looking to improve run time. I will follow up with the Nested Model Co-Chairs to make sure they are on board with this recommendation and, if so, we can implement this fix in GEOS-Chem 13.3.0.

But,  I used 0.25x0.3125, GEOS_FP nested-grid. The default setting of this grid is 300/600s. 
Will the accuracy of the simulation be reasonable even if the time step is doubled(600/1200s)?

**Comment by msulprizio** [comment]
@dorosieun That's right. The recommendation has always been to use the 300/600s timesteps when using 0.25x0.3125 GEOS-FP nested-grid simulations because of the higher grid resolution.

Here is also a wiki page with some tips for speeding up your simulations: http://wiki.seas.harvard.edu/geos-chem/index.php/Speeding_up_GEOS-Chem.
